cypress/e2e/audio-player/audio-player.spec.ts

```diff
@@ -0,0 +1,563 @@
+        uploadFile("cypress/fixtures/1sec-long-name-audio-file.ogg");
+
+        cy.get(".mx_RoomView_MessageList").within(() => {
+            // Assert the audio player is rendered
+            cy.get(".mx_EventTile_last .mx_AudioPlayer_container").should("exist");
+
+            cy.get(".mx_EventTile_last")
+                .realHover()
+                .within(() => {
+                    // Click "Reply" button on MessageActionBar
+                    cy.get('[aria-label="Reply"]').click({ force: false });
+                });
+        });
+
+        // Reply to the player with another audio file
+        uploadFile("cypress/fixtures/1sec.ogg");
+
+        cy.get(".mx_RoomView_MessageList").within(() => {
+            cy.get(".mx_EventTile_last").within(() => {
+                // Assert that replied audio file is rendered as file button inside ReplyChain
+                cy.get(".mx_ReplyChain_wrapper").within(() => {
+                    cy.get(".mx_MFileBody_info[role='button']").within(() => {
+                        // Assert that the file button has file name
+                        cy.get(".mx_MFileBody_info_filename").should("exist");
+                    });
+                });
+
+                cy.get(".mx_MAudioBody").within(() => {
+                    // Assert that the play button is visible
+                    cy.get("[data-testid='play-pause-button'][aria-label='Play']").should("be.visible");
+                });
+            });
+        });
+
+        // Scroll to the bottom to make the audio player visible for Percy tests
+        cy.get(".mx_MainSplit .mx_ScrollPanel").scrollTo("bottom");
+
+        cy.get(".mx_RoomView_MessageList").within(() => {
+            const takeSnapshotReply = (layout: string) => {
+                // Click the timestamp to highlight the event tile on which player is rendered
+                cy.get(".mx_EventTile_last .mx_MessageTimestamp").click();
+                cy.get(".mx_EventTile.mx_EventTile_selected .mx_MAudioBody").should("be.visible");
+
+                cy.get(".mx_EventTile_last .mx_MAudioBody").should("be.visible");
+                cy.get(".mx_EventTile_last").percySnapshotElement(
+                    `"EventTile of audio player with a reply on ${layout} layout"`,
+                    { percyCSS },
+                );
+                cy.log(`"Took a snapshot of EventTile of audio player with a reply on ${layout} layout"`);
+            };
+
+            // Take a snapshot of EventTile with a reply on group layout
+            cy.setSettingValue("layout", null, SettingLevel.DEVICE, Layout.Group);
+            takeSnapshotReply("group");
+
+            // Take a snapshot of EventTile with a reply on bubble layout
+            cy.setSettingValue("layout", null, SettingLevel.DEVICE, Layout.Bubble);
+            takeSnapshotReply("bubble");
+        });
+    });
+
+    it("should create ReplyChain with multiple audio files", () => {
```

Comment:
```suggestion
    it("should support creating a ReplyChain with multiple audio files", () => {
```

Reply:
I think I remember but there are multiple (two or three) elements named `mx_ReplyChain` inside one Reply Chain. I'll check again and get back to you.

Reply:
This was addressed with https://github.com/matrix-org/matrix-react-sdk/pull/10441/commits/3742b4f1abdce292c05e1bf40241a187d5e1cf06.